tests/integration_tests/build/test_binary_szie.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Test that checks if the release binary size is within expected size"""
+
+import os
+import pytest
+
+import host_tools.cargo_build as host
+
+BINARY_SIZE_TARGET = 3137280
+"""Target binary size to be checked in bytes"""
+
+BINARY_SIZE_DELTA = 1024 * 1024 / 2
+"""Tolerance of 0.5 MB (expressed in bytes) allowed for binary size"""
+
+FEATURES = [""]
+
+@pytest.mark.parametrize(
+    "features",
+    FEATURES
+)
+@pytest.mark.timeout(500)
+def test_binary_size(test_session_root_path, features):
+    """
+    This will check if the size of the release binary is within expected range
+    """
+
+    fc_binary, _ = host.get_firecracker_binaries(
```

Comment:
Does this rebuild the binaries when the first microvm is instantiated?

Reply:
Done